inst/@sym/chebyshevT.m

```diff
@@ -20,6 +20,8 @@
 %% @documentencoding UTF-8
 %% @deftypefn  {Function File} {@var{s} =} chebyshevT (@var{n}, @var{x})
 %% Find the nth symbolic Chebyshev polynomial of the first kind.
+%% If n is a vector then it returns a vector with Chebyshev polynomials
+%% of the first kind for each element of 'n'.
```

Comment:
1.  There needs to be a blank line after the first sentence (for `help chebyshevT`).

2.  This should be written `If @var{n} is a vector then it returns...`